resources/assets/components/CampaignBanner/CampaignBanner.js

```diff
@@ -175,7 +175,10 @@ const CampaignBanner = ({
             }
           >
             {!isAffiliated ? (
-              <div className={`pt-6 ${!groupType ? 'w-2/3 sm:w-1/2' : null}`}>
+              <div
+                data-testid="campaign-banner-affiliated"
```

Comment:
How's about something like 
```suggestion
                data-testid="scholarship-modal-signup-button"
```

To complement the `campaign-banner-signup-button` above?